package.json

```diff
@@ -16,6 +16,7 @@
     "@expo/vector-icons": "^13.0.0",
     "@expo/webpack-config": "^0.17.2",
     "@react-native-async-storage/async-storage": "~1.17.3",
+    "@react-native-community/checkbox": "^0.5.14",
```

Comment:
Are we using this?

Reply:
I forgot to uninstall it. I'll remove it from the dependecies.